components/pages/header.vue

```diff
@@ -1,13 +1,281 @@
 <template>
-  <h1>Header</h1>
+  <header class="header">
+    <nav class="header__nav">
+      <div class="header__left">
+        <ul class="header__menu">
+          <li
+            v-for="(item, index) in menuList"
+            :key="index"
+            class="header__menu__item"
+          >
+            <a href="/">{{ item.label }}</a>
+          </li>
```

Comment:
Plz add click event when this item is clicked.
```suggestion
          <li
            v-for="(item, index) in menuList"
            :key="index"
            class="header__menu__item"
            @click="handleClickEvent(item)"
           >
            <a href="/">{{ item.label }}</a>
          </li>
```


```
methods: {
  handleClickEvent(item) {
    TODO handle redirect page
  }
}
```